### ISSUE TITLE
The picture of network is not same as your code?

### ISSUE THREAD

**Issue by hdmjdp** [fatchord/WaveRNN#26]
In picture, your last 2 dense layer has skip connection, but your code does not has.

**Comment by fatchord** [comment]
@hdmjdp I don't see the discrepancy you point out. There are three dense layers at the end and the last two flow directly through without concat or residual between them.

**Comment by hdmjdp** [comment]
@fatchord 
                 x = x + h2
                x = torch.cat([x, a3_t], dim=1)
                x = F.relu(self.fc1(x))
                
                x = torch.cat([x, a4_t], dim=1)
                x = F.relu(self.fc2(x))
                x = self.fc3(x)


in your picture, there is a skip operation("+") between the second dense layer and the third dense layer. But your code did not has .

**Comment by fatchord** [comment]
@hdmjdp If you take a look at the diagram again you will see three dense layers of which there is only a skip connection between the first (fc1) and the second (fc2). Not sure what I can do to convince you!

**Comment by hwong39** [comment]
@hdmjdp I have the same question with you. 
@fatchord As you said "there is only a skip connection between the first (fc1) and the second (fc2)", but i did not find a skip connection between the first (fc1) and the second (fc2) in your NB4b.

**Comment by fatchord** [comment]
@hwong39 There's a connection coming from the resnet into fc2. 

Anyway it turns out my 'throw in everything but the kitchen sink' approach to this model has made it more complicated than it needs to be. I'm currently training a much simpler version and will upload soon.

**Comment by hdmjdp** [comment]
@fatchord “ a much simpler version and will upload soon.”， this version can run in realtime in cpu. I expected it very much.
